doc/site/content/docs/guides/yardmaps/_index.md

```diff
@@ -52,14 +52,40 @@ Basic characters:
 | u      | ❌                   | ✅                  | ❌                     | ✅                    | ❌         | ❌        |
 | e      | ❌                   | ⚠️ exit only        | ❌                     | ⚠️ exit only          | ❌         | ❌        |
 
-Others:
-
-- `h` has a special meaning - it **has to be the first letter and means a high resolution yardmap**,
-  i.e. there have to be 4x more characters and each will correspond to a "regular" 8x8 elmo game square instead of the larger 16x16 elmo footprint square.
-- `w`, `x`, `f`: deprecated, same as `o`.
-- **whitespace is ignored**, which you can use to neatly **arrange the rectangle visually**.
+<dl>
+	<dt>Active/Inactive</dt>
+	<dd>
+		Whether <code>YARD_OPEN</code> is true or false (controlled by animation,
+		default false)
+	</dd>
+	<dt>Needs geo</dt>
+	<dd>
+		Whether the unit can only be built on top of terrain marked as geothermal.
+	</dd>
+</dl>
+
+A few remarks:
+
+- **whitespace is ignored**, which you can use to neatly **arrange the
+  rectangle visually**.
 - anything else is ignored as well but may be used in the future.
```

Comment:
Could be confusing to say "anything else" before all options were mentioned (`h` was previously above this line but is now below and in a different section).